openlibrary/components/IdentifiersInput.vue

```diff
@@ -115,40 +115,48 @@ export default {
     },
 
     computed: {
-        popularEditionConfigs: function() {
-            if (this.edition_popular) {
-                const popularConfigs = JSON.parse(decodeURIComponent(this.edition_popular));
-                return Object.fromEntries(popularConfigs.map(e => [e.name, e]));
+        parsedConfigs: function() {
+            return JSON.parse(decodeURIComponent(this.id_config_string))
+        },
+        popularIdDict: function() {
+            if (this.output_selector.includes('Edition')) {
```

Comment:
With a `popular_ids` property, we can also avoid checking for Edition here as well, and just return the popular config if popular IDs are supplied.